packages/data-provider/src/controllers/submittedDataController.ts

```diff
@@ -118,6 +119,77 @@ const controller = (dependencies: BaseDependencies) => {
 				next(error);
 			}
 		},
+
+		getSubmittedDataByQuery: async (
+			req: Request<{ categoryId: string; organization: string }, {}, any, { page: string; pageSize: string }>,
+			res: any,
+			next: any,
+		) => {
+			try {
+				const categoryId = Number(req.params.categoryId);
+				const organization = req.params.organization;
+				const sqon = req.body;
+				let page = parseInt(req.query.page as string) || defaultPage;
+				let pageSize = parseInt(req.query.pageSize as string) || defaultPageSize;
```

Comment:
I don't see these beinf redefined anywhere. Should be const?

Reply:
changed to `const` as we're not reassigning these variables